cmdtest.go

```diff
@@ -691,6 +691,25 @@ func fixedArgBuiltin(nargs int, f func([]string) ([]byte, error)) CommandFunc {
 	}
 }
 
+func rename(oldpath, newpath string) error {
+	// NOTE: This is used in place of os.Rename because it's
+	// able to rename files between different mount points.
+	src, err := os.Open(oldpath)
+	if err != nil {
+		return err
+	}
+	dst, err := os.Create(newpath)
+	if err != nil {
+		return err
+	}
+	defer dst.Close()
```

Comment:
You can't ignore Close errors on write. Close may cause bytes to be written as part of a final flush.